app/services/doc_auth/lexis_nexis/responses/true_id_response.rb

```diff
@@ -125,11 +127,6 @@ def billed?
           !!doc_auth_result
         end
 
-        def doc_type_supported?
```

Comment:
This is a good refactor, but I'm hesitant to include it in this PR. Consider adding a followup ticket and a comment here instead?

Reply:
Have to change the implementation anyway to include country code, so take the chance to using the same logic for both assureid and trueid.